src/DynamoCore/Models/DynamoModelCommands.cs

```diff
@@ -597,12 +597,15 @@ private void UpdateModelValueImpl(UpdateModelValueCommand command)
             WorkspaceModel targetWorkspace = CurrentWorkspace;
             if (!command.WorkspaceGuid.Equals(Guid.Empty))
                 targetWorkspace = Workspaces.FirstOrDefault(w => w.Guid.Equals(command.WorkspaceGuid));
-
-            if (targetWorkspace != null)
+            try
             {
-                targetWorkspace.UpdateModelValue(command.ModelGuids,
+                targetWorkspace?.UpdateModelValue(command.ModelGuids,
                     command.Name, command.Value);
             }
+            catch (Exception ex)
+            {
+                Logger.LogError(ex.Message);
```

Comment:
Do we know what kinds of exceptions are these and why are they thrown?

Reply:
There are code inside the function throw the exception, but they are not caught anywhere, so Dynamo will crash